app/controllers/casa_cases_controller.rb

```diff
@@ -35,6 +35,11 @@ def new
   end
 
   def edit
+    if current_role == "Volunteer"
+      @siblings_casa_cases = current_user.casa_cases.excluding(@casa_case)
+    elsif ["Supervisor", "Casa Admin"].include? current_role
```

Comment:
ideally we would find a way to use our pundit policies for permissions work like this

Reply:
Yeah it's surely not the best way to do :)
I'm still looking for a way to do it with pundit